src/utils/types/zetkin.ts

```diff
@@ -500,3 +500,20 @@ export type {
   ZetkinViewColumn,
   ZetkinViewRow,
 };
+
+export type ZetkinCreatePerson = {
```

Comment:
What is the purpose of this type? The name (with the `Zetkin` prefix) and the module it's in (`zetkin.ts`) implies that it's a type related to the Zetkin API, but it is not. What is the benefit of having this type compared to the regular `ZetkinPerson` type?

Reply:
Oh yeah that is true. I could use ZetkinPerson..!